resources/client/client.ts

```diff
@@ -2,9 +2,26 @@ import ClientUtils from './cl_utils';
 import { ResourceConfig } from '../../typings/config';
 
 // Setup and export the config for the resource
-export const config: ResourceConfig = JSON.parse(
-  LoadResourceFile(GetCurrentResourceName(), 'config.json'),
-);
+export const config: ResourceConfig = (() => {
+  const config = JSON.parse(LoadResourceFile(GetCurrentResourceName(), 'config.json'));
+  const convars = ['general', 'phoneAsItem'];
```

Comment:
Any reason we don't just define all the sections here?

Reply:
Those two sections could be useful for another resource to modify, such as QB/ESX changing the convar value for Config.General.Multichar and setting the phone item export.

I was under the impression that you didn't want everything set as convars.